frontend/app/services/raoidc-service.server.ts

```diff
@@ -0,0 +1,113 @@
+/**
+ * RAOIDC Service is a service module responsible for managing authentication with RAOIDC.
+ */
+import { createHash, subtle } from 'node:crypto';
+import { ProxyAgent, fetch as undiciFetch } from 'undici';
+import { toNodeReadable } from 'web-streams-node';
+
+import { privateKeyPemToCryptoKey, publicKeyPemToCryptoKey } from '~/utils/crypto-utils.server';
+import { getEnv } from '~/utils/env.server';
+import { getLogger } from '~/utils/logging.server';
+import { type ClientMetadata, type FetchFunctionInit, fetchAccessToken, fetchServerMetadata, fetchUserInfo, generateAuthorizationRequest, generateCodeChallenge, generateRandomState } from '~/utils/raoidc-utils.server';
+
+const log = getLogger('raoidc-service.server');
```

Comment:
Based on the changes you've made to other services, the logger initialisation should be in the `createRaoidcService()` function.

Reply:
[Fixed](https://github.com/DTS-STN/canadian-dental-care-plan/blob/6aa29d67733c2c166a18a8d3ac17f691bee9be51/frontend/app/services/raoidc-service.server.ts#L16)